WooCommerce/src/main/kotlin/com/woocommerce/android/ui/analytics/AnalyticsViewModel.kt

```diff
@@ -190,6 +193,10 @@ class AnalyticsViewModel @Inject constructor(
         }
     }
 
+    fun onTrackableUIInteraction() {
+        usageTracksEventEmitter.interacted()
+    }
```

Comment:
np: Nothing that needs to be addressed but this could also be written as: 

```suggestion
    fun onTrackableUIInteraction() = usageTracksEventEmitter.interacted()
```

Reply:
Good point! Thanks! Fixed here: https://github.com/woocommerce/woocommerce-android/pull/7682/commits/74a6165bfc2c51ab3292057fae38c43150c3aef2